src/main/java/edu/byu/ece/rapidSmith/design/subsite/PropertyList.java

```diff
@@ -20,32 +20,36 @@
 
 package edu.byu.ece.rapidSmith.design.subsite;
 
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
 
 /**
  * This class represents an objects that can have properties. Any class that needs <br> 
  * to have properties, should inherit from this class. 
  * 
  * @author Thomas Townsend
```

Comment:
Remove the reference me as the author. I am trying to do this whenever I can since it is meaningless documentation.